lib/smart_app_launch/client_suite/access_bsca_interaction_test.rb

```diff
@@ -13,6 +13,13 @@ class SMARTClientAccessBackendServicesConfidentialAsymmetricInteraction < Infern
       During this test, Inferno will wait for the client to access data
       using a SMART token obtained using the Backend Services flow.
     )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@229',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@233',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@235',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@236',
```

Comment:
@236 seems a better fit for 5.11 (already there). Remove from here.

@236
[when requesting] an access token via HTTP POST to the FHIR authorization server’s token endpoint URL [the] `grant_type` [parameter is] `required` [and SHALL contain the] … Fixed value: `client_credentials`